stage1/init/init.go

```diff
@@ -185,30 +185,17 @@ func stage1() int {
 	}
 
 	mirrorLocalZoneInfo(c.Root)
-	c.MetadataServiceURL = common.MetadataServicePublicURL()
```

Comment:
I think I even made you add a note to that effect in networking.md ... sorry for the noise